packages/context-menu/src/vaadin-contextmenu-items-mixin.js

```diff
@@ -268,8 +268,9 @@ export const ItemsMixin = (superClass) =>
           // Close the menu
           this.close();
           this.listenOn.focus();
-        } else if (key === 'Tab') {
-          // Close all menus
+        } else if (key === 'Tab' && !event.defaultPrevented) {
+          // Close all menus unless the Tab key was handled separately
+          // which is the case e.g. in menu-bar with Tab navigation.
```

Comment:
This was previously fired on submenu item Tab and then due to `requestAnimationFrame()` the menu was re-opened. Added a check for `defaultPrevented` which is now used when switching submenu item.